django_lightweight_queue/backends/reliable_redis.py

```diff
@@ -101,6 +101,28 @@ def processed_job(self, queue, worker_number, job):
     def length(self, queue):
         return self.client.llen(self._key(queue))
 
+    def deduplicate(self, queue):
```

Comment:
Docs on the return value would be cool - just as the early-exit of `return 0, 0` isn't super clear :).